### PR TITLE
fix missing json tags

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/assign @openshift/openshift-team-developer-productivity-test-platform

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
https://github.com/openshift/release/pull/9260

File: test/ci-operator-integration/multi-stage/configs/master/openshift-hyperkube-master.yaml [hunk 1]
```diff
@@ -72,7 +72,7 @@ tests:
       credentials:
         - namespace: "ns"
           name: "name"
-          mountPath: "/tmp"
```
Comment (alvaroaleman) [inline test/ci-operator-integration/multi-stage/configs/master/openshift-hyperkube-master.yaml:75]: This isn't in use yet?

/lgtm
/approve
/hold

Comment (stevekuznetsov) [reply test/ci-operator-integration/multi-stage/configs/master/openshift-hyperkube-master.yaml:75]: It is :(

**Pull request merged**